packages/react-aria-components/scripts/create-react-aria-components/index.js

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/env node
+
+import chalk from 'chalk';
+import {copyComponents} from './helpers/copyComponents.js';
+import {copyIndexFile} from './helpers/copyIndexFile.js';
+import {copyPackageJson} from './helpers/copyPackageJson.js';
+import {exec} from 'child_process';
+import path from 'path';
+import prompts from 'prompts';
+import {readComponents} from './helpers/readComponents.js';
+import {readTemplates} from './helpers/readTemplates.js';
+
+async function main() {
+  const {action} = await prompts({
+    type: 'select',
+    name: 'action',
+    message: 'What would you like to do?',
+    choices: [
+      {
+        title: 'Create a new component library',
+        value: 'Create a new library'
+      },
+      {
+        title: 'Add component(s) to an existing project',
+        value: 'Add components'
+      }
+    ]
+  });
+
+  if (action === 'Add components') {
+    console.log('Components will be added to the current directory.');
+  }
+
+  let projectName;
+  if (action === 'Create a new library') {
+    const response = await prompts({
+      type: 'text',
+      name: 'projectName',
+      message: 'What is your project named?',
+      initial: 'my-component-library'
+    });
+    projectName = response.projectName;
+  } else {
+    projectName = '.';
+  }
+
+  const templates = await readTemplates();
+  const {template} = await prompts({
+    type: 'select',
+    name: 'template',
+    message: 'Select a template',
+    choices: templates.map((t) => ({title: t, value: t}))
+  });
+
+  const components = await readComponents(template);
+  let {selectedComponents} = await prompts({
+    type: 'multiselect',
+    name: 'selectedComponents',
+    message: 'Select the components you want to include',
+    choices: [
+      {title: 'All (Default)', value: 'All', selected: true},
```

Comment:
Feels weird to have a "All" option that can be selected alongside the other options IMO, can probably omit it since there is already a "toggle all" shortcut

Reply:
Same feed back for the features prompt

Reply:
Good idea, I'll see if this can be improved. At the least we could just auto select all of them and remove the "All" option, which might be a little more clear.